cairo-verifier/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use std::fs;
+
+use clap::Parser;
+use crate::utils::error::VerifierError;
+
+mod commands;
+mod utils;
+
+#[derive(Parser)]
+struct Cli {
+    proof_path: String,
+}
+
+fn main() -> Result<(), VerifierError> {
+    let args: commands::ProverArgs = commands::ProverArgs::parse();
+
+    let res = match args.entity {
+        commands::ProverEntity::Verify(args) => {   
+            utils::verify_proof(&args.proof_path)
+        },
+        commands::ProverEntity::Prove(args) => {
```

Comment:
Prove emits a proof file based on a cairo execution generating
- a trace
- a memory "trace" ?

Reply:
These come from the cairo execution in the browser ?